websrc/aardvark_renderer/src/aardvark_traverser.ts

```diff
@@ -215,9 +215,9 @@ export class AvDefaultTraverser
 		this.m_endpoint.registerHandler( MessageType.LostEndpoint, this.onLostEndpoint );
 	}
 
-	@bind onEndpointOpen()
+	@bind onEndpointOpen(settings: AvRendererConfig)
 	{
-
+		Av().renderer.setRendererConfig(settings)
```

Comment:
I'm glad this bit of this was easy, at least. :)

Reply:
That was super easy :P